tools/tslint-rules/noHostDecoratorInConcreteRule.ts

```diff
@@ -0,0 +1,36 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+export class Rule extends Lint.Rules.AbstractRule {
+    apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
```

Comment:
It looks like the indentation of the whole file is too large (seems like it's a 4, but it should be 2).